### ISSUE TITLE
How to train embedding network

### ISSUE THREAD

**Issue by jinde-liu** [aharley/segaware#5]
I have found that the code only contains the part for test and no code for training. Is the embedding network trained separately from deeplab network?

**Comment by aharley** [comment]
Yes: I recommend pre-training the embeddings using VOC 2012, or COCO. You can also train them end-to-end in the full pipeline. 
I provided a config file for an embedding network.
